### ISSUE TITLE
validation loss can be NaN if the last batch contains only one sample

### ISSUE THREAD

**Issue by MoritzPalm** [MoritzPalm/LiDAR_Detection#1]
if the batch contains only one sample, it is apparently possible that no priors are positive, preventing the calculation of the L1 loss and the cross entropy loss.
This is not because this one image has no positive labels, but the exact reasons still need investigation, eg. it is yet unclear if this only happens with one sample or if any underfilled batches could lead to this issue.

Current workaround: enabling drop_last in the dataloaders to prevent underfilled batches
